src/ot_croissant/crumbs/distribution.py

```diff
@@ -29,26 +29,6 @@ def get_metadata(self):
         """Return the distribution metadata."""
         return self.distribution
 
-    def generate_distribution_description(self, id: str) -> str:
```

Comment:
This function had to be moved from the distribution to the recordset class, as the tags are not part of the distribution description.